samples/grids/grid/editing-excel-style-custom/src/index.ts

```diff
@@ -0,0 +1,83 @@
+import 'igniteui-webcomponents-grids/grids/combined';
+import { ComponentRenderer, WebGridDescriptionModule } from 'igniteui-webcomponents-core';
+import { IgcGridComponent } from 'igniteui-webcomponents-grids/grids';
+import { NwindData } from './NwindData';
+
+import "igniteui-webcomponents-grids/grids/themes/light/bootstrap.css";
+
+import "./index.css";
+
+export class Sample {
+
+    private grid1: IgcGridComponent
+    private _bind: () => void;
+
+    constructor() {
+        var grid1 = this.grid1 = document.getElementById('grid1') as IgcGridComponent;
+    
+        this._bind = () => {
+            grid1.data = this.nwindData;
+            grid1.addEventListener('keydown', (event: KeyboardEvent) => {
+                var key = event.key; // Correct variable name used
+                const activeElem = grid1.selectedCells[0];
+
+                if ((key >= '0' && key <= '9') || (key.toLowerCase() >= 'a' && key.toLowerCase() <= 'z')) {
+        
+                    if (activeElem && activeElem.editMode === false) {
+                        activeElem.setEditMode(true);
+                        activeElem.value = key;
+                    }
+                }
+    
+                if (key === 'Enter') {
+
+                    var nextRow = this.getNextEditableRowIndex(activeElem.row.index, activeElem.row.cells, event.shiftKey);
```

Comment:
This does not look correct. The second parameter expects the grid dataview, not the cells.